prody/proteins/interactions.py

```diff
@@ -0,0 +1,2043 @@
+# -*- coding: utf-8 -*-
+
+"""This module defines functions for calculating different types of interactions 
+in protein structure, between proteins or between protein and ligand.
+The following interactions are available for protein interactions:
+        (1) Hydrogen bonds
+        (2) Salt Bridges
+        (3) Repulsive Ionic Bonding 
+        (4) Pi stacking interactions
+        (5) Pi-cation interactions
+        (6) Hydrophobic interactions
+
+For protein-ligand interactions (3) is replaced by water bridges.
+"""
+
+__author__ = 'Karolina Mikulska-Ruminska'
+__credits__ = ['James Krieger','Karolina Mikulska-Ruminska']
+__email__ = ['karolami@pitt.edu', 'KRIEGERJ@pitt.edu']
```

Comment:
it's probably better not to use our pitt emails when we probably won't have them that much longer. We could use my gmail maybe

Reply:
Changed